.github/workflows/scripts/check_for_redirects.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  /**
+   * Add redirects-needed label if count of deleted files is greater than 0
+   *
+   * @param {Object} obj.artifactName - Name of artifiact file to check
+   */
+  addRedirectsNeededLabel: async ({ github, context, fs, artifactName }) => {
+    const {
+      payload: {
+        repository: {
+          owner: { login: ownerLogin },
+          name: repoName
+        }
+      }
+    } = context;
+
+    const artifactContents = fs
+      .readFileSync(`./${artifactName}.txt`)
+      .toString();
+
+    const [prNumberStr, numOfDeletedFilesStr] = artifactContents.split('\n');
+
+    console.log('PR number that triggered workflow: ', prNumberStr);
+    console.log('Number of deleted files: ', numOfDeletedFilesStr);
```

Comment:
Nit:
```suggestion
    console.log('PR number that triggered workflow:', prNumberStr);
    console.log('Number of deleted files:', numOfDeletedFilesStr);
```

node will put whitespace between the two anyway.